packages/core/src/components/Alert/Alert.scss

```diff
@@ -22,20 +22,25 @@ Markup:
 Style guide: components.alert
 */
 
-$alert-icon-size: $spacer-5;
+$h-padding: $spacer-2;
+$bar-size: $spacer-1;
+$alert-icon-size: $spacer-7;
```

Comment:
This icon size feels a little too big, in relation to the size of the heading and body text. I have a slight preference for bumping this down. Thoughts on reverting it back to `$spacer-5`? That would align the icon with the first line of text:

![image](https://user-images.githubusercontent.com/371943/29639719-bc225b80-8829-11e7-920e-7448705d0673.png)

![image](https://user-images.githubusercontent.com/371943/29639732-c8618664-8829-11e7-8284-43c1a657d73d.png)

Reply:
Agreed, I like that better.